### PR TITLE
feat(components): [color-picker] add `no-trigger` prop

### PR DESCRIPTION
**Pull request by zhixiaotong**
Please make sure these boxes are checked before submitting your PR, thank you!

Added a way to use the color panel directly.

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by zhixiaotong** [comment]
https://github.com/element-plus/element-plus/assets/34681550/9c4e032b-03fb-4cdf-9121-bb56a54d1316

@btea

**Comment by zhixiaotong** [comment]
欸，为啥docs build失败，本地是成功的 - . -

**Comment by zhixiaotong** [comment]
> 欸，为啥docs build失败，本地是成功的 - . -

`type check`问题，但回退到dev分支打包也报错误，设置null恢复正常

**Comment by zhixiaotong** [comment]
@btea 打搅了，我看到有的admin平台有类似这种取色器效果，不需要通过触发器，如图

image

其实，除了`color-picker`，日期选择也是类似的，也有类似的使用场景（比如工作台的日历效果），但目前`ep`还不支持直接展示。这个`pr`提交已经过去两周了，但没有人对这个pr有看法或建议，哪怕是负面的，所以...我只能主动一点了。:)

**Comment by btea** [comment]
Sorry, I've been busy lately. I'll review it later.

File: packages/components/color-picker/src/color-picker.ts [hunk 1]
```diff
@@ -70,6 +70,10 @@ export const colorPickerProps = buildProps({
     type: Boolean,
     default: true,
   },
+  noTrigger: {
+    type: Boolean,
+    default: false,
```
Comment (btea) [inline packages/components/color-picker/src/color-picker.ts:75]: The default value can be omitted here.

File: packages/components/color-picker/src/color-picker.vue [hunk 2]
```diff
@@ -1,5 +1,72 @@
 <template>
+  <DefineTemplate>
+    <div
+      :aria-disabled="noTrigger ? colorDisabled : undefined"
```
Comment (btea) [inline packages/components/color-picker/src/color-picker.vue:4]: Regarding the parameters or classes configured with the `disabled` attribute, there should be no need to first determine `noTrigger`.
```suggestion
      :aria-disabled="colorDisabled || undefined"
```

**Comment by zhixiaotong** [comment]
> Sorry, I've been busy lately. I'll review it later.

哈哈，无事，我比较在意这个需求对于ep是否合理，实现是否满足你们的想法。如果这个方式可以的话，我后面也会对datepicker添加`noTrigger`

**Comment by btea** [comment]
cc @element-plus/backers

**Comment by zhixiaotong** [comment]
> Sorry, I've been busy lately. I'll review it later.

稍后修复😊

File: packages/components/color-picker/src/color-picker.vue (hunk 2 above)
Comment (zhixiaotong) [reply packages/components/color-picker/src/color-picker.vue]: done

File: packages/components/color-picker/src/color-picker.ts (hunk 1 above)
Comment (zhixiaotong) [reply packages/components/color-picker/src/color-picker.ts]: done

**Comment by btea** [comment]
@zhixiaotong Hello, thank you for your contribution! 

Through discussions with other people and designers, this does not conform to the design specifications, and this may not be what the component is supposed to do. 

If you really need this functionality, it may be better to fork the code and customize it yourself. ❤️

**Comment by zhixiaotong** [comment]
> @zhixiaotong Hello, thank you for your contribution!
> 
> Through discussions with other people and designers, this does not conform to the design specifications, and this may not be what the component is supposed to do.
> 
> If you really need this functionality, it may be better to fork the code and customize it yourself. ❤️

okay...
虽然...
但是...其他UI库也有这方面的能力。我觉得ep也需要吧？
https://www.naiveui.com/zh-CN/light/components/date-picker#panel.vue
https://arco.design/vue/component/date-picker#panel-only

**Comment by zhixiaotong** [comment]
Anyway, I respect the team's discussion results.

**Comment by stupidream** [comment]
我也觉得这个功能很有用，为什么就不加呢，不知道有什么样设计规范，我也是看到很多ui库都有这样的能力。

**Pull request closed without merging**